### PR TITLE
HPCC-13106 Expand GetLogicalFileAttribute Docs

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@AttilaVamos Please review

### CODE REVIEW

File: docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml [hunk 1]
```diff
@@ -31,7 +31,10 @@
           <entry><emphasis>attrname</emphasis></entry>
 
           <entry>A null-terminated string containing the name of the file
-          attribute to return.</entry>
+          attribute to return. Possible values are recordSize, recordCount,
+          size, clusterName, directory, numparts, owner, description, ECL,
+          partmask, numparts, name, modified, or protected. The value is
+          case-sensitive.</entry>
```
Comment (AttilaVamos) [inline docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml]: And 'format', 'job', 'checkSum', 'kind', 'csvSeparate', 'csvTerminate' and 'workunit'

**Comment by JamesDeFabia** [comment]
@AttilaVamos Edits made,  please look again

File: docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml [hunk 2]
```diff
@@ -31,7 +31,11 @@
           <entry><emphasis>attrname</emphasis></entry>
 
           <entry>A null-terminated string containing the name of the file
-          attribute to return.</entry>
+          attribute to return. Possible values are recordSize, recordCount,
+          size, clusterName, directory, numparts, owner, description, ECL,
+          partmask, numparts, name, modified, format, job, checkSum, kind,
+          csvSeparate, csvTerminate, headerLength, footerLength, rowTag,
+          workunit, or protected. The value is case-sensitive.</entry>
```
Comment (AttilaVamos) [inline docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml]: I found some more: 
- accessed
- maxRecordSize
- csvQuote (for CSV file if it is defined for the command)
- blockCompressed, compressedSize (if the file compressed) 

The 'numparts' is duplicated:

**Comment by AttilaVamos** [comment]
@JamesDeFabia Back to you.

**Comment by JamesDeFabia** [comment]
@AttilaVamos  Edits made, back to you

File: docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml [hunk 3]
```diff
@@ -31,7 +31,12 @@
           <entry><emphasis>attrname</emphasis></entry>
 
           <entry>A null-terminated string containing the name of the file
-          attribute to return.</entry>
+          attribute to return. Possible values are recordSize, recordCount,
+          size, clusterName, directory, owner, description, ECL, partmask,
+          numparts, name, modified, format, job, checkSum, kind, csvSeparate,
+          csvTerminate, headerLength, footerLength, rowTag, workunit,
+          accessed, maxRecordSize, csvQuote, blockCompressed, compressedSize,
+          or protected. The value is case-sensitive.</entry>
```
Comment (AttilaVamos) [inline docs/ECLStandardLibraryReference/SLR-Mods/GetLogicalFileAttribute.xml]: A couple of new:
fileCrc
formatCrc

**Comment by richardkchapman** [comment]
@JamesDeFabia Back to you...

**Comment by JamesDeFabia** [comment]
@AttilaVamos Back to you

**Comment by AttilaVamos** [comment]
@JamesDeFabia It seems OK.

**Pull request merged**